Sources/Descriptions.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  Created by Jesse Squires
+//  https://www.jessesquires.com
+//
+//  Documentation
+//  https://jessesquires.github.io/ReactiveCollectionsKit
+//
+//  GitHub
+//  https://github.com/jessesquires/ReactiveCollectionsKit
+//
+//  Copyright © 2019-present Jesse Squires
+//
+
+import Foundation
+
+/// :nodoc:
```

Comment:
Small nit: we can remove all the `/// :nodoc:` because this has no effect for `private` members. 😊